pkg/generator/pool.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/ipfs/go-log"
+	"github.com/ipfs/go-log/v2"
```

Comment:
Shouldn't we replace it in *all* files?

Reply:
Not necessarily, `v2` is backward compatible so we can use it in places where needed.